system_services/rover-jetson-raman.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=MRover Jetson Raman Spectroscope Service
+
+[Service]
+EnvironmentFile=/etc/default/rover.env
+User=root
+ExecStart=/usr/bin/socat tcp-listen:8888,reuseaddr,fork file:/dev/raman,nonblock,waitlock=/var/run/tty0.lock,b115200,raw,echo=0
```

Comment:
i'll just leave this as a review but i dont expect u to solve this issue atm but this will only work if the /dev/raman is defined, which it's not

Reply:
this is what i'm talking about https://github.com/umrover/mrover-workspace/blob/main/ansible/roles/jetson_services/files/99-usb-serial.rules

Reply:
What's supposed to go in /dev/raman?

Reply:
Its like a USB representing the nucleo. The info is in the 99 USB serial rules